private_gpt/settings/settings.py

```diff
@@ -92,6 +92,7 @@ class LocalSettings(BaseModel):
     llm_hf_repo_id: str
     llm_hf_model_file: str
     embedding_hf_model_name: str
+    llm_hf_resume_download: bool
```

Comment:
Suggesting to make it default to `False` if no values are provided in documentation
```suggestion
    llm_hf_resume_download: bool = False
```

Reply:
Has been removed from settings.